app/src/main/java/org/schabi/newpipe/player/playqueue/ChannelPlayQueue.java

```diff
@@ -39,9 +32,9 @@ public void fetch() {
             ExtractorHelper.getChannelInfo(this.serviceId, this.baseUrl, false)
                     .subscribeOn(Schedulers.io())
                     .observeOn(AndroidSchedulers.mainThread())
-                    .subscribe(getHeadListObserver());
+                    .subscribe((Consumer<? super ChannelInfo>) getHeadListObserver());
```

Comment:
I don't think I did this part correctly and I'm not sure what this even is supposed to do, though I don't think it breaks anything when I remove it.